tests/test_distributions.py

```diff
@@ -197,7 +199,11 @@ def test_single():
         distributions.UniformDistribution(low=1.0, high=1.001),
         distributions.LogUniformDistribution(low=7.3, high=10),
         distributions.DiscreteUniformDistribution(low=-30, high=-20, q=2),
+        distributions.DiscreteUniformDistribution(low=-30, high=-20, q=10),
+        # In Python, "0.3 - 0.2 != 0.1" is True.
+        distributions.DiscreteUniformDistribution(low=0.2, high=0.3, q=0.1),
```

Comment:
We could add one more case. But it might be overkill, please feel free to ignore it.
```suggestion
        distributions.DiscreteUniformDistribution(low=0.2, high=0.3, q=0.1),
        distributions.DiscreteUniformDistribution(low=0.7, high=0.8, q=0.1),
```
since
```
>>> 0.8 - 0.7
0.10000000000000009
>>> 0.3 - 0.2
0.09999999999999998
```